### ISSUE TITLE
Thermal controller broke the game

### ISSUE THREAD

**Issue by Redstylt** [mekanism/Mekanism#5750]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*
Checked

#### Issue description:
The game get into a "not responding" state after placing a thermal controller

#### Steps to reproduce:
1.create the structure of an evaporation plant
2.place a controller
3.game is broken ! (no loot anymore / stay on "save the game" when quitting)

#### Version (make sure you are on the latest version before reporting):
**Forge:** 30.0.35
**Mekanism:** 9.9.1
**Other relevant version:** MC 1.15.1

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

No crash log because game isn't really crashed.... more like an infinite code loop

**Comment by pupnewfster** [comment]
Without a crash log or the latest.log there is not much we can do given the fact that I have a thermal evaporation controller that works just fine in my test world.

**Comment by Redstylt** [comment]
There is nothing in the latest.log... But i took screenshots to reproduce the steps
https://imgur.com/a/G7tIeCt

**Comment by Discomanco** [comment]
Getting the same issue in 9.9.1, fresh client Mek+JEI. The game just stops when placing the Controller, and neither the latest.log or debug.log gets to put anything. Forge 30.0.36
Debug:
```
[10Jan2020 21:34:18.475] [Render thread/INFO] [mezz.jei.util.LoggedTimer/]: Starting JEI took 1.310 s
[10Jan2020 21:34:18.645] [Render thread/INFO] [Mekanism/]: Received Cardboard Box blacklist entries from server (0 explicit blocks, 0 mod wildcards)
[10Jan2020 21:34:18.655] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] [Mekanism] Warning: Mekanism is currently in alpha, and is not recommended for widespread use in modpacks. There are likely to be game breaking bugs, and various other issues that you can read more about here.
[10Jan2020 21:34:18.656] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 42 advancements
[10Jan2020 21:34:53.044] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] [Debug]: Chunk borders: shown
[10Jan2020 21:35:35.020] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Saving and pausing game...
[10Jan2020 21:35:35.041] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'Test'/minecraft:overworld
[10Jan2020 21:35:35.211] [Server thread/DEBUG] [net.minecraftforge.fml.FMLWorldPersistenceHook/WP]: Gathering id map for writing to world save Test
[10Jan2020 21:35:41.213] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Discomanco> Placing Controller now.
[10Jan2020 21:35:41.215] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] <Discomanco> Placing Controller now.
[10Jan2020 21:36:34.278] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Stopping!
```

Latest.log:
```
[10Jan2020 21:34:18.475] [Render thread/INFO] [mezz.jei.util.LoggedTimer/]: Starting JEI took 1.310 s
[10Jan2020 21:34:18.645] [Render thread/INFO] [Mekanism/]: Received Cardboard Box blacklist entries from server (0 explicit blocks, 0 mod wildcards)
[10Jan2020 21:34:18.655] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] [Mekanism] Warning: Mekanism is currently in alpha, and is not recommended for widespread use in modpacks. There are likely to be game breaking bugs, and various other issues that you can read more about here.
[10Jan2020 21:34:18.656] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 42 advancements
[10Jan2020 21:34:53.044] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] [Debug]: Chunk borders: shown
[10Jan2020 21:35:35.020] [Server thread/INFO] [net.minecraft.server.integrated.IntegratedServer/]: Saving and pausing game...
[10Jan2020 21:35:35.041] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'Test'/minecraft:overworld
[10Jan2020 21:35:41.213] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: <Discomanco> Placing Controller now.
[10Jan2020 21:35:41.215] [Render thread/INFO] [net.minecraft.client.gui.NewChatGui/]: [CHAT] <Discomanco> Placing Controller now.
[10Jan2020 21:36:34.278] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Stopping!
```

**Comment by pupnewfster** [comment]
I will try to look into it again/build another evaporation plant once I am done fixing what I currently am doing.

**Comment by pupnewfster** [comment]
Ok, so I was able to reproduce this, and the odd thing is it only seems to happen when the controller is not on the layer of the multiblock. Which is why my test world worked fine as out of habbit I put my controller at the top. I will try to look into this more tomorrow and see if I can figure out where it is stalling.

**Comment by pupnewfster** [comment]
Fixed with the release of 9.9.2
